app/admin/categories.rb

```diff
@@ -8,26 +8,57 @@
   index do
     id_column
     column :name
+    column :short_name
     column :description
+    column "Parent Category" do |p|
+      if !p.nil?
+        Category.find_by_id(p.parent_category_id)
+       end
+    end
     column :related_terms
+    column :is_other
     actions
   end
 
   show do
     attributes_table do
       row :id
       row :name
+      row :short_name
       row :description
+      row "Parent Category" do |p|
+        if !p.nil?
+          parent_cat = Category.find_by_id(p.parent_category_id)
+        end
+      end
       row :related_terms
+      row :is_other
+
+      category_practice = CategoryPractice.where(category_id: params["id"])
```

Comment:
I don't think we need this `lines 36 -46`

Reply:
Cool.  I see that you removed those lines.  thx